dataactbroker/handlers/fileHandler.py

```diff
@@ -1343,6 +1344,10 @@ def list_submissions(page, limit, certified, sort='modified', order='desc'):
     """ List submission based on current page and amount to display. If provided, filter based on
     certification status """
     sess = GlobalDB.db().session
+    db = GlobalDB.db()
+    metadata = MetaData(bind=db.engine)
+
+    submission_updated_view = Table('submission_updated_at_view', metadata, autoload=True, info=dict(is_view=True))
```

Comment:
When dealing with views in SQL Alchemy, can we query it the same way as the other models?